packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -1629,3 +1629,32 @@ async fn test_transaction_script_workflow() {
     let response = call_handler.get_response(receipts).unwrap();
     assert_eq!(response.value, 42);
 }
+
+#[tokio::test]
+async fn nested_enums() {
+    abigen!(
+        MyContract,
+        "packages/fuels-abigen-macro/tests/test_projects/nested_enums/out/debug/nested_enums-abi.json"
+    );
+
+    let wallet = launch_provider_and_get_single_wallet().await;
+
+    let id = Contract::deploy(
+        "tests/test_projects/nested_enums/out/debug/nested_enums.bin",
+        &wallet,
+        TxParameters::default(),
+    )
+    .await
+    .unwrap();
+
+    let instance = MyContract::new(id.to_string(), wallet.clone());
+    let expected = Option::Some(Identity::Address(Address::zeroed()));
+
+    let result = instance.some_address().call().await.unwrap();
```

Comment:
use `?`

Reply:
As far as I can tell, all test use this pattern. I would not like to change it only in my test. If we want to change it in all test, I would gladly do it and I suggest we make a new issue.